packages/@textlint/source-code-fixer/README.md

```diff
@@ -0,0 +1,56 @@
+# @textlint/source-code-fixer
+
+Apply @textlint/kernel lint result to text.
+
+Almost user just user `@textlint/kernel`'s `fixText` API instead of This package.
```

Comment:
`fixText` is a stable API.


If you want to get fixed results, please just use kernel API.